addons/web/static/src/core/signature/name_and_signature.js

```diff
@@ -123,6 +122,19 @@ export class NameAndSignature extends Component {
         this.props.signature.isSignatureEmpty = this.isSignatureEmpty;
     }
 
+    /**
+     * Load a base64 encoded image, as dataURL, into the signature field.
+     * Update the isSignatureEmpty flag to reflect the correct state.
+     * This prevents the flag from incorrectly staying true when a signature
+     * image is already passed in the props.
+     */
+    fromDataURL(dataURL) {
```

Comment:
The original `fromDataURL` method (from signature_pad library) actually has 2 arguments (dataURL and options), so to keep the same interface, both should be forwarded.
The simpler and more future-proof way to do it is to forward all arguments (cf. original implementation).